src/components/channels_list.js

```diff
@@ -40,8 +40,7 @@ class ChannelsList extends Component {
     componentWillReceiveProps(props) {
         if (props.channels.current_channel_id &&
           !this.props.channels.current_channel_id) {
-            console.warn('TODO route to PostsList'); // eslint-disable-line no-console
-            // TODO: Routes.PostLists(); // eslint-disable-line new-cap
+            Routes.PostsList(); // eslint-disable-line new-cap
```

Comment:
This is just part of the react-native-router-flux api.  It comes from the routes.js file.

```
                    <Scene
                        key='PostsList'
                        component={PostsList}
                        title='Posts List'
                    />
```

We can change the key to be camel case.  Perhaps `goToPostsList`